File: backend/app/Models/PriceListItem.php
<?php

namespace App\Models;

use Illuminate\Database\Eloquent\Model;
use App\Traits\HasLog;

class PriceListItem extends Model
{
    use HasLog;
    protected $fillable = ['price_list_id', 'item_id', 'price'];

    protected $casts = [
        'price' => 'decimal:2',
    ];

    public function priceList()
    {
        return $this->belongsTo(PriceList::class);
    }

    public function item()
    {
        return $this->belongsTo(Item::class);
    }
}

File: compose-app/shared/src/commonMain/kotlin/com/sales/app/domain/model/PriceListItem.kt
package com.sales.app.domain.model

import kotlinx.serialization.Serializable

@Serializable
data class PriceListItem(
    val id: Long,
    val priceListId: Long,
    val itemId: Long,
    val price: Double,
    val itemName: String? = null,
    val itemCode: String? = null,
    val standardPrice: Double? = null
)
